### PR TITLE
test-Oceane-Dan

### PR DESCRIPTION
**Pull request by dandan-l**
In the work progress of adding dates to the "hahaha" message, uncomment the sqlite3 library and added Pandas library.

### CODE REVIEW

File: favorite_built_in_modules.py [hunk 1]
```diff
@@ -27,6 +28,7 @@ def main():
     #
     ###########################################################################
     print("John DiMatteo likes datetime:", datetime.__doc__)
+    print("hahhaha",datetime.date)
```
Comment (peleyal) [inline favorite_built_in_modules.py:31]: This is funny! Good change 👍

**Review by dandan-l — COMMENTED** [review]
@jdimatteo  review my code, please

**Review by jdimatteo — APPROVED** [review]
Cool, we both like datetime!

**Review by peleyal — APPROVED** [review]
Change looks good to me. Did you test it? 😉

**Pull request merged**